package.json

```
@@ -62,6 +65,7 @@
     "react-icons": "5.0.1",
     "react-redux": "9.1.0",
     "react-router-dom": "6.22.2",
+    "swiper": "^11.1.1",
```

```suggestion
    "swiper": "11.1.1",
```
kindly use exact version for consistency with the rest of the dependencies